packages/evolution-generator/src/scripts/generate_section_configs.py

```diff
@@ -58,14 +58,13 @@ def generate_section_configs(excel_file_path: str):
         # Iterate through each row in the sheet, starting from the second row
         for row_number, row in enumerate(rows[1:], start=2):
             # Get values from the row
-            (
-                section,
-                title_fr,
-                title_en,
```

Comment:
Hmm, I'm not sure if we should proceed with that. The get_values_from_row function was a good way to raise an error if "Error getting values from row" occurred.

Can you please remind me of the specific case that was causing issues? Perhaps we can work on fixing the get_values_from_row function.

Reply:
If the sheet had more columns than the ones expected, it would fail (like the case where I removed the groups from the section config, we absolutely need to update the files to remove this columns, which is not ideal).

The `get_headers` function above made sure the expected headers are there, this one should raise an error if one columns is missing. But if there is more information in the file, it should not raise anything. This commit fixes that.

Reply:
Okay, sounds good.